pkgs/misc/rivalcfg/default.nix

```diff
@@ -0,0 +1,19 @@
+{ stdenv, fetchurl, python3Packages }:
+
+with python3Packages; buildPythonPackage rec {
+  pname = "rivalcfg";
+  version = "3.6.0";
+
+  meta = with stdenv.lib; {
+    description = "Small CLI utility program that allows you to configure SteelSeries Rival gaming mice";
+    license     = licenses.wtfpl;
+  };
+
+  src = fetchurl {
```

Comment:
I think using fetchFromGitHub is preferred

Reply:
correct, although it will resolve mostly to the same thing.